.changeset/shiny-otters-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
```

Comment:
According to [semantic versioning](https://semver.org/), a backward-compatible bug fix should be a "patch":

```suggestion
'@sumup/circuit-ui': patch
```